amundsen_application/static/js/components/TableDetail/ColumnStats/tests/index.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { ColumnStats, ColumnStatsProps } from '../';
+
+
+
+describe('ColumnStats', () => {
+  const setup = (propOverrides?: Partial<ColumnStatsProps>) => {
+    const props = {
+      stats: [
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "count", stat_val: "12345" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "count_null", stat_val: "123" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "count_distinct", stat_val: "22" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "count_zero", stat_val: "44" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "max", stat_val: "1237466454" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "min", stat_val: "856" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "avg", stat_val: "2356575" },
+        { end_epoch: 1571616000, start_epoch: 1571616000, stat_type: "stddev", stat_val: "1234563" },
+      ],
+      ...propOverrides,
+    };
+    const wrapper = shallow<ColumnStats>(<ColumnStats {...props} />);
+    return { props, wrapper };
+  };
+
+  const { wrapper, props } = setup();
+  const instance = wrapper.instance();
+
+  describe('formatDate', () => {
+    it('formats a date in the correct format', () => {
```

Comment:
nit: Why two similar tests? Does one test some weird edge case?

Reply:
I can delete one case. I just get the feeling that date formats are easily broken so they need additional testing :)